src/state/loanReport/actions.js

```diff
@@ -52,35 +52,35 @@ export function updateLoanReport(payload) {
 }
 
 /**
- * Create an action to set current pair.
- * @param {string[]} pairs pairs
+ * Create an action to set current symbols.
+ * @param {string[]} symbols symbols
```

Comment:
I think it's enough `symbols` one time so no need to describe it with just name again

Reply:
going to fix that in a separate PR since there are many such cases